source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -189,8 +189,10 @@ Further to the NORMAL Pad Mode, 2 additional layers of functionality, SHIFT & AL
    Transport", "Sample 1-8"
    ":hwlabel:`SHIFT`
 
+   Secondary Mode
+
    • (LED On)", "Hot Cue 9-16", "Roll Loop", "Fader Cuts 8", "Sample 9-16"
-   ":hwlabel:`Hold` or :hwlabel:`Double Click`
+   ":hwlabel:`Hold` or :hwlabel:`2X PRESS`
```

Comment:
Somehow I find the hwlabels confusing for this. I'll propose another version..

Reply:
I updated all the suggestions.